packages/fast-jss-manager/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@microsoft/fast-jss-manager",
+  "version": "1.0.0",
+  "description": "Provides common code to all jss-manager implementations",
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "author": {
```

Comment:
package.json should be consistent with how we have done other package.json.  So the Author should match.